finder-frontend/config/deploy.rb

```diff
@@ -12,7 +12,7 @@
 set :db_config_file, false
 
 namespace :deploy do
-  task :cold do
-    puts "There's no cold task for this project, just deploy normally"
+  task :update_registries_cache do
```

Comment:
Does something need to be added to make this task get called?  Or does that happen automatically because it's in the `deploy` namespace?

Reply:
Yes I think one needs to call it with a before/after hook to shoehorn it into the deploy lifecycle.

After [finalize_update](https://github.com/capistrano/capistrano-2.x-docs/blob/master/Capistrano-Tasks.md#deployfinalize_update) seems like the done thing.